test/ReactWrapper-spec.js

```diff
@@ -372,7 +372,7 @@ describeWithDOM('mount', () => {
 
     });
 
-    it('should not find components with invalid attributes', () => {
+    it.skip('should not find components with invalid attributes', () => {
```

Comment:
The point of this test is to test on purpose wrong attributes that should be ignored by enzyme. I'm not sure there's anything to do. Since I don't like warnings when running tests, I would do:
```
const _warn = console.warn;
console.warn = () => {};
// tests
console.warn = _warn;
```